images/keycloak/tests/main.tf

```diff
@@ -42,6 +42,11 @@ resource "helm_release" "test" {
   })]
 }
 
+data "oci_exec_test" "keycloak-production-test" {
+  digest = var.digest
+  script = "${path.module}/01-keycloak-production-mode.sh"
```

Comment:
The numbering scheme was only necessary under a different testing strategy, you can just name this without the 01.

Reply:
Good to know, thanks for calling out @imjasonh - amended as suggested :)